### PR TITLE
Removed php5 version checks

### PR DESCRIPTION
**Pull request by acrobat**
I am targeting this branch, because support for all version below php7.1 are removed on 3.x.

## Changelog

```markdown
### Removed
- Removed php5 version checks
```

## Subject

Some cleanup to remove php5 version checks

### CODE REVIEW

File: DependencyInjection/SonataPageExtension.php [hunk 1]
```diff
@@ -499,73 +495,4 @@ public function configurePageServices(ContainerBuilder $container, array $config
         $definition = $container->getDefinition('sonata.page.page_service_manager');
         $definition->addMethodCall('setDefault', [new Reference($config['default_page_service'])]);
     }
-
-    /**
-     * Add class to compile.
-     */
-    public function configureClassesToCompile()
```
Comment (core23) [inline DependencyInjection/SonataPageExtension.php]: Technical, it's a BC break. Not sure if we could accept this, 'cause this method should never been called in this context

Comment (greg0ire) [reply DependencyInjection/SonataPageExtension.php]: It's fine.

**Comment by greg0ire** [comment]
Thanks @acrobat !

**Comment by OskarStark** [comment]
minor?

**Comment by acrobat** [comment]
I would say so because of the php and symfony version bump. It's possible to release this as a patch version but imo it's cleaner to this in a minor release

**Pull request merged**